package.json

```
@@ -11,6 +11,8 @@
     "docs:serve": "cd ./docs && yarn serve",
     "docs:deploy": "cd ./docs && yarn deploy",
     "lint": "./scripts/lint",
+    "format:scss": "find ./src -type f -name '*.scss' | xargs yarn prettier --write ",
+    "format:md": "yarn prettier --write '**/*.md'",
```

```suggestion
    "format:md": "prettier --write '**/*.md'",
```
I don't think we need the `yarn` here.